.dependency-lock/pom.xml

```diff
@@ -3356,12 +3624,55 @@
             <dependency-lock:integrity>sha512:AaXqb1tDvwlMUqUOGDJaYK97sC500k+bwscn1D5RRXj9losw/yL50nIMrsBxRY+f+C0Roh+7HrxC2CA+c3xLUg==</dependency-lock:integrity>
           </dependency>
         </dependencies>
+        <dependency-lock:integrity>sha512:S6jb9yYTd3YXcqHG11OYLWrPEPYE979T4gS2qJrTruTx8JhI3RveUNTLuhfD7UpFl3Co8OhSQ8MNIRKDTLd/Lw==</dependency-lock:integrity>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
```

Comment:
Plugin added to be able to attach test resources (for testing and IDE integration)